### PR TITLE
fix: minor tweaks to vordr slack messages

### PR DESCRIPTION
**Pull request by omBratteng**
Adds shared post title and source name to slack vordr post messages

### CODE REVIEW

File: src/workers/vordrPostPrevented.ts [hunk 1]
```diff
@@ -23,6 +23,11 @@ export const vordrPostPrevented: TypedWorker<'api.v1.post-visible'> = {
 
       const author = await post.author;
       const scout = await post.scout;
+      const source = await post.source;
```
Comment (capJavert) [inline src/workers/vordrPostPrevented.ts:26]: preload post in relations?

Comment (omBratteng) [reply src/workers/vordrPostPrevented.ts:27]: done 🙈

File: src/common/slack.ts [hunk 2]
```diff
@@ -135,16 +137,17 @@ export const notifyNewVordrPost = async (
     text: 'New post prevented by vordr',
     attachments: [
       {
-        title: post.title!,
+        title: sharedPost ? (post.title ?? sharedPost.title) : post.title,
```
Comment (capJavert) [inline src/common/slack.ts:140]: would it be more simple to say `post.title ?? sharedPost?.title`?

**Review by capJavert — APPROVED** [review]

File: src/common/slack.ts (hunk 2 above)
Comment (omBratteng) [reply src/common/slack.ts:140]: Probably, I did these changes without running it locally. Could try it like that
```suggestion
        title: post.title ?? sharedPost?.title,
```

Comment (capJavert) [reply src/common/slack.ts]: You want title and if no title try shared title, else its empty either way? @omBratteng

Comment (omBratteng) [reply src/common/slack.ts]: Yeah

**Review by rebelchris — APPROVED** [review]

**Pull request merged**